armi/tests/detailedAxialExpansion/armiRun.yaml

```diff
@@ -0,0 +1,47 @@
+metadata:
```

Comment:
Please add a unit test or two that use these files, before merging them into the repo.

Reply:
Addressed.

https://github.com/terrapower/armi/pull/657/commits/cdf1aed4b09b0d57a39c7e0aee68a975fccb43ec